components/sup/src/manager/service/health.rs

```diff
@@ -189,8 +191,20 @@ pub fn check_repeatedly(supervisor: Arc<Mutex<Supervisor>>,
                                          password.clone()).await;
 
             let interval = if result == HealthCheckResult::Ok {
-                // routine health check
-                nominal_interval
+                if !first_successful_health_check {
+                    // If this was the first successful check, splay future health check runs across
+                    // the nominal interval
+                    let splay = rand::thread_rng().gen_range(0, u64::from(nominal_interval));
+                    let splay = Duration::from_secs(splay);
+                    debug!("Adding {}s of splay for {} health-checks",
+                           splay.as_secs(),
+                           service_group);
+                    first_successful_health_check = true;
+                    splay.into()
```

Comment:
Reading the log message I would have expected to see `nominal_interval + splay.into()` here.  In the end it doesn't matter, just a minor point and I don't have a great suggestion for better wording

Reply:
Yeah, I thought making the health check interval shorter than the configured interval was more acceptable than making it longer.

Agreed the debug message was confusing. [cd76d3](https://github.com/habitat-sh/habitat/pull/7850/commits/cd76d3b712298347c1c9cf41d276f2b75593095e) hopefully improves it.